components/logins/README.md

```diff
@@ -15,11 +20,69 @@ The relevant directories are as follows:
   editing logins using the code in `src`.
 - `ffi`: The Rust public FFI bindings. This is a (memory-unsafe, by necessity)
   API that is exposed to Kotlin and Swift. It leverages the `ffi_support` crate
-  to avoid many issues and make it more safe than it othrwise would be. At the
+  to avoid many issues and make it more safe than it otherwise would be. At the
   time of this writing, it uses JSON for marshalling data over the FFI, however
   in the future we will likely use protocol buffers.
 - `android`: This contains android bindings to logins, written in Kotlin. These
   use JNA to call into to the code in `ffi`.
 - `ios`: This contains the iOS binding to logins, written in Swift. These use
   Swift's native support for calling code written in C to call into the code in
   `ffi`.
+
+## Features
+1. Locally encrypted storage of username and password information
+1. Create, Update and Delete (CRUD) operations for login data
+1. Syncing of logins for devices and apps connected via Firefox Accounts
+1. Import functionality from existing login storage (ex: Fx Desktop or Fennec)
```

Comment:
Nit: Let's use Firefox for iOS as an example. We don't import from Desktop (yet!), and you've covered Fennec with data migration below!